src/main/kotlin/model/Card.kt

```diff
@@ -0,0 +1,3 @@
+package model
+
+data class Card(val rank: Rank, val suite: Suite, val faceUp: Boolean = true)
```

Comment:
```suggestion
package model

/**
 * Blah Blah Blah
 */
data class Card(val rank: Rank, val suite: Suite, val faceUp: Boolean = true)
```

I think the most reliable, simple, and developer-friendly way is still leaving a comment.
As long as it includes enough context, it’ll help prevent confusion for anyone reading the code later. 👍 

like this
<img width="589" height="148" alt="image" src="https://github.com/user-attachments/assets/e0a4e634-7353-4f79-8fa2-edd94568930e" />